app/controllers/api/service_dialogs_controller.rb

```diff
@@ -30,8 +30,8 @@ def create_resource(_type, _id, data)
     def edit_resource(type, id, data)
       service_dialog = resource_search(id, type, Dialog)
       begin
-        service_dialog.update_tabs(data['content']['dialog_tabs']) if data['content']
-        service_dialog.update!(data.except('content'))
+        service_dialog.update!(data.except('dialog_tabs', 'content'))
+        service_dialog.update_tabs(data['dialog_tabs'] || data['content']['dialog_tabs']) if data['dialog_tabs'] || data['content']
```

Comment:
What if both are specified, what is the expectation.

specs needed to test both signatures.

/cc @jntullo

Reply:
If both are specified, the one without 'content' takes precendence. 
Maybe warning about it?

Reply:
I think we should log a warning, maybe mentioning something like "Both 'dialog_tabs' and 'content' : 'dialog_tabs' were specified. 'content' : 'dialog_tabs' will be ignored."